LayoutTests/imported/w3c/web-platform-tests/html/cross-origin-embedder-policy/block-local-documents-inheriting-none.https-expected.txt

```diff
@@ -1,3 +1,3 @@
 
-FAIL Prevent local scheme documents from loading within a COEP: require-corp iframe if they inherit COEP: none assert_equals: expected 3 but got 0
+FAIL Prevent local scheme documents from loading within a COEP: require-corp iframe if they inherit COEP: none assert_equals: expected 3 but got 1
```

Comment:
Some progress, I guess .... :-)

Reply:
Yes, I'll investigate remaining failures in a follow-up.

Reply:
This check that we're getting 1 report for blob, 1 for data and 1 for about:blank. We're only getting the one for blob and not the others. I suspect this is a generic COEP issue and not a reporting issue but I'll investigate further.